envoy/go.mod

```
@@ -1,6 +1,6 @@
 module github.com/cybozu/neco-containers/envoy
 
-go 1.21
+go 1.22
```

I guess we need specify the patch version.

```suggestion
go 1.22.2
```

---

@masa213f  Wny do you think so?
My understandig is that it is treated equally with `x.xx.0` when we don't specify the patch.

---

I faced a build error a while ago without the "go toolchain" directive.
And I commented that I wondered if it would be the same this time.
If not, that's fine.

---

I googled, and it seems that we should specify the patch version after `1.21.x`.
https://github.com/golang/go/issues/62278

I think this can be built with 1.22.x so, I'll specify the version `1.22.0`.